internal/command/flags.go

```diff
@@ -55,6 +56,10 @@ func installEventingOption(targetCmd *cobra.Command) {
 	targetCmd.Flags().BoolVar(&installEventing, "install-eventing", false, "install Eventing on quickstart cluster")
 }
 
+func installCamelOption(targetCmd *cobra.Command) {
+	targetCmd.Flags().BoolVar(&installCamel, "install-camel", false, "install Apache Camel K on quickstart cluster")
```

Comment:
I would prefer something like `--extensions camel`. That could potentially server for additional 3rd party installations. But not a hard opinion, we probably won't grow this list to unmanageable state.

Reply:
I think that is a good idea for a future improvement. Sounds good?

Reply:
I'll create a new issue from this suggestion. All good for the scope of this PR.